src/Model/IssueModel.php

```diff
@@ -27,6 +28,13 @@ class IssueModel
      */
     private $body;
 
+    /**
+     * @var DateTime
+     * @Serializer\Type("DateTime")
```

Comment:
If you know the format of `created_at` the JMS\Serializer can automatically parse the date string into an actual object.

See the documentation here: https://jmsyst.com/libs/serializer/master/reference/annotations#type
For this (as mentioned above) use `DateTimeImmutable<...>` as the type.